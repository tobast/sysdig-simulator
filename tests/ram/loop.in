3
001001010
001000101
000000000
