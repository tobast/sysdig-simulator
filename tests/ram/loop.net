INPUT ra, we, wa, c
OUTPUT o
VAR
  l : 4, c : 4, o : 4, ra : 2, wa : 2, we
IN
l = OR o c
o = RAM 2 4 ra we wa l
