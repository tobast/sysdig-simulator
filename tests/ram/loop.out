0000
1010
1111
